apps/pie-microsite/src/_includes/scripts.njk

```diff
@@ -2,3 +2,33 @@
 <script defer src="/assets/js/accordions.js" type="module"></script>
 <script defer src="/assets/js/toggled-content.js" type="module"></script>
 <script defer src="/assets/js/prism.js" type="module"></script>
+<script>
+    console.log(`
+______ _____ _____
+| ___ \\_   _|  ___|
+| |_/ / | | | |__
+|  __/  | | |  __|
+| |    _| |_| |___
+\\_|    \\___/\\____/
+`);
+
+console.log(`
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@,*,@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@**,*******,***@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@.*********************,@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@(*,,*******,,,,,,,,,,,,,,@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@.*********,,,,,,,,,,,,,,,,,,,,,@@@@@@@@@@@@@@@@@@@@@@
+@@@@@***,,***,,,,,,,,,,,,,,,,,,,,,,,,,,,@@@@@@@@@@@@@@@@@@@@
+@@@@@*****,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,, @@@@@@@@@@@@@@@@@@
+@@@@@@@@@@,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@@@@@@@@@@@@@
+@@@@@@@@@@,/////*,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@@@@@@@
+@@@@@@@@@@,/////////////*,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@@@@
+@@@@@@@@@@,////////////////////////*,,,,,,,,,,,,,,,*//,,, @@
+@@@@@@@@@@,*///////////////////////////////////////*,,,,,,,@
+@@@@@@@@@@,,,,*/////////////////////////////////////////%@@@
+@@@@@@@@@@@@@@@@@@@,,,,,,*////////////////////////////,@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ ,,,,,////////////////*@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@.,,,,,///,@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@`)
```

Comment:
😎

Reply:
In light of this utterly devastating comment, I have refactored it